opentelemetry-semantic-conventions/scripts/generate-consts-from-spec.sh

```diff
@@ -6,7 +6,7 @@ CRATE_DIR="${SCRIPT_DIR}/../"
 
 # freeze the spec version and generator version to make generation reproducible
 SPEC_VERSION=1.27.0
-SEMCOVGEN_VERSION=0.25.0
+WEAVER_VERSION=v0.9.2
```

Comment:
To clarify, the process for generating the code remains unchanged, correct? Specifically, we still increment the spec/weaver version here and run this script as usual?

Reply:
Yes correct.